src/docs/testing/index.md

```diff
@@ -87,15 +85,18 @@ dev_dependencies:
     sdk: flutter
 ```
 
-(This is needed even if your test does not itself explicitly import
-`flutter_test`, because the test framework itself uses it behind the
-scenes.)
+**Note:** The dev dependency on `flutter_test` is required even if
+your test doesn't explicitly import `flutter_test`,
+because the test framework uses `flutter_test` behind the scenes.
```

Comment:
Nit: Could you wrap this tip text in `{{site.alert.note}}` ... `{{site.alert.end}}`? (Search the site files for examples.)